pkg/rc/replication_controller.go

```diff
@@ -937,7 +939,6 @@ func (rc *replicationController) scheduleNewNodeForTransfer(rcFields fields.RC,
 	ok, _, err := auditingTransaction.Commit(rc.txner)
 	switch {
 	case err != nil:
-		logger.WithError(err).Errorln("could not schedule new node")
```

Comment:
I moved this log to where `scheduleNewNodeForTransfer` is called so that other errors returned in this function would also trigger a log.